### ISSUE TITLE
Single Motor Unit Action Potential Abnormality

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5937]
**Preferred term label:** Single Motor Unit Action Potential Abnormality

**Synonyms** SMUP Abnormality

**Definition (free text, please give PubMed ID)** Abnormally high or low single motor unit action potential reading (-Pk Amps)

**Parent term (use hpo.jax.org/app)** EMG abnormality HP:0003457

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
